src/styles/normalize.scss

```
@@ -0,0 +1,9 @@
+body {
+  margin: 0;
+  font-family: Arial, sans-serif;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vw;
```

vw - it's  viewport width)
```suggestion
  height: 100vh;
```